src/main/java/uk/gov/hmcts/divorce/document/print/model/Print.java

```diff
@@ -12,4 +12,6 @@ public class Print {
     private final String caseId;
     private final String caseRef;
     private final String letterType;
+    private final List<String> recipients;
```

Comment:
Consider removing the @AllArgsConstructor and creating the list in the constructor

    public Print(final List<Letter> letters, final String caseId, final String caseRef, final String letterType, final String recipientName) {
        this.letters = letters;
        this.caseId = caseId;
        this.caseRef = caseRef;
        this.letterType = letterType;
        this.recipients = List.of(caseId, recipientName, letterType);
    }